docs/cloud/http-api.md

```diff
@@ -182,6 +182,25 @@ So if the delete query matches a branch, **every** series under that branch will
 curl -u "api_key:<Your Grafana.com API Key>" https://<tsdbgw>/metrics/delete -d query=some.series.to.delete.*
 ```
 
+#### Tagged With `/tags/delSeries`
+
+Deletes metrics which match the `path` parameter(s).
+
+* Method: POST
+* API key type:
```

Comment:
Not sure what the `write` value is here, but a lot of the other `API key type` fields in this document are blank as well.

Reply:
I'm also not that familiar with how the authentication works. 

For example the `/metrics` endpoint here also has `"write"` as the first parameter to `a.GenerateHandlers()`: https://github.com/raintank/tsdb-gw/blob/master/cmd/tsdb-gw/main.go#L189
In the doc the specified `API key type` for `/metrics` is `any (including Viewer)`. Seems strange to me that a viewer could post metrics into `/metrics`, so i think either that doc is wrong or this should also be `any (including Viewer)`.

Do you know that @woodsaj

Reply:
in tsdb-gw you can configure whether roles are enforced.
that setting defaults to false, we have never enforced roles. meaning if the api key is allowed, the role of it is ignored.

Reply:
Updated docs to match the aspirational state of our system?

It now claims to require the `Editor` role to delete metrics 72b9dfa7cdfbf97a29a53f3552a5680eef960ca8.